src/javascript/JContent/ContentTree/ContentTree.jsx

```diff
@@ -150,11 +151,24 @@ export const ContentTree = ({setPathAction, openPathAction, closePathAction, ite
     let contextualMenu = useRef();
 
     return (
-        <React.Fragment>
+        <div style={{
+            display: 'flex',
+            flexDirection: 'column',
+            alignItems: 'stretch',
+            justifyContent: 'stretch',
+            height: '100%'
```

Comment:
Still not possible, percentage height is calculated based on the specified parent height - but the parent height is undefined. In the case of secondary-nav, the ContentTree is used inside an AccordionItem which has a flex height (so, calculated but unspecified). It might work on some browsers, but not everywhere, and differently. I'm not sure it's possible to correctly handle that without adding a display:flex on the parent (here, accordionItem)

Reply:
Moonstoine 2.5.9 has been released with flexCol on accordion item content